apps/devhub/templates/devhub/docs/policies-contact.html

```diff
@@ -42,10 +42,9 @@ <h4 id="section-security">{{ _('Add-on Security Vulnerabilities') }}</h4>
 <h4 id="section-website">{{ _('Website Functionality & Development') }}</h4>
 <p>
 {% trans %}
-If you've found a problem with the site, we'd love to fix it. Please <a href="https://bugzilla.mozilla.org/enter_bug.cgi?product=addons.mozilla.org">file a bug report</a> in Bugzilla, including the location of the problem and how you encountered it.
+If you've found a problem with the site, we'd love to fix it. Please <a href="https://github.com/mozilla/olympia/issues">file a bug report</a> in GitHub, including the location of the problem and how you encountered it.
```

Comment:
This could link straight to `issues/new` but maybe we want them to search first.